src/pages/task/TestsTableCore.tsx

```diff
@@ -77,17 +81,60 @@ const columns: Array<ColumnProps<TaskTestsData>> = [
   },
   {
     title: "Time",
-    width: "25%",
+    width: "20%",
     dataIndex: "duration",
     key: Categories.Duration,
     sorter: true,
     render: (text: number): string => {
       const ms = text * 1000;
       return msToDuration(Math.trunc(ms));
     }
+  },
+  {
+    title: "Logs",
+    width: "20%",
+    dataIndex: "logs",
+    key: "logs",
+    sorter: false,
+    render: ({
+      htmlDisplayURL,
+      rawDisplayURL
+    }: {
+      htmlDisplayURL: string;
+      rawDisplayURL: string;
+    }): JSX.Element => {
+      return (
+        <>
+          {htmlDisplayURL && (
+            <ButtonWrapper>
+              <Button
+                className="htmlBtn"
```

Comment:
This className is applied so cypress can find this button

Reply:
what about using ids for cypress. we would only use ids for testing which makes it clear it's not for styling